### PR TITLE
chore: zendframework/zend-diactoros -> laminas/laminas-diactoros

### PR DESCRIPTION
**Pull request by Gounlaf**
* update zendframework/zend-diactoros -> laminas/laminas-diactoros
* also fix psr-4

Signed-off-by: Levis Florian <levis.florian@gmail.com>

### CODE REVIEW

File: tests/phpunit.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	convertErrorsToExceptions="true" convertNoticesToExceptions="false"
 	convertWarningsToExceptions="false">
 	<testsuite name="UnitTests">
-		<directory>./unit</directory>
+		<directory>./Unit</directory>
```
Comment (mcustiel) [inline tests/phpunit.xml]: The directory name starts with a lowercase u, this change is not needed.

File: composer.json [hunk 2]
```diff
@@ -31,9 +31,9 @@
     "license" : "GPL-3.0-or-later",
     "require" : {
         "php" : "^7.2",
-        "zendframework/zend-diactoros" : "^2.0",
-        "guzzlehttp/guzzle" : "^6.0"
-    }, 
+        "guzzlehttp/guzzle" : "^6.0",
+        "laminas/laminas-diactoros": "^2.2"
```
Comment (mcustiel) [inline composer.json:35]: Thanks, this was in my todo list.

File: composer.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
     },
     "autoload-dev" : {
         "psr-4" : {
-            "Mcustiel\\Phiremock\\Test\\Unit\\" : "tests/unit"
+            "Mcustiel\\Phiremock\\Tests\\Unit\\" : "tests/unit"
```
Comment (mcustiel) [inline composer.json:25]: Good point. Thanks.

File: tests/phpunit.xml (hunk 1 above)
Comment (Gounlaf) [reply tests/phpunit.xml]: Fixed

**Review by mcustiel — APPROVED** [review]

**Pull request merged**